layouts/page/about.html

```diff
@@ -119,7 +119,7 @@ <h2 class="text-center md:text-left">Recent News</h2>
                             <div class="pl-4 flex-1">
                                 <div class="text-sm text-gray-500">{{ .date }}</div>
                                 <h5 class="mt-2 mb-0">{{ .title }}</h5>
-                                <a class="block my-2" href="{{ .url }}">Read more &rarr;</a>
+                                <a data-track="news-{{ .url | urlize }}" class="block my-2" href="{{ .url }}">Read more &rarr;</a>
```

Comment:
I'd expect this just to be `news`, or `news-read-on`, or something, rather than `news-https//www.businessinsider.com/18-hottest-enterprise-startups-from-seattles-booming-tech-scene-2019-12`.

Reply:
My thought here was from a historical perspective if the links change then we would want to record the target URL. Just so our ids to reflect changes in content.